content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported.
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].
+
+**Note**: Some specific builds will work with Datadog profiling if they support [Java Flight Recorder][2].
+
+## Getting Started
+
+To begin profiling applications written in any JVM based runtime:
+
+1. Download `dd-java-agent.jar`, which contains the Agent class files:
+
+    ```shell
+    wget -O dd-java-agent.jar 'https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.datadoghq&a=dd-java-agent&v=LATEST'
+    ```
+
+2. Add the following JVM argument when starting your application in your IDE, Maven or Gradle application script, or `java -jar` command:
+
+    ```shell
+    -javaagent:<PATH>/dd-java-agent.jar
+    ```
+
+3. Enable profiling by adding `-Ddd.profiling.enabled=true` to the JVM command line (in addition to any existing APM Agent parameters).
+
+4. Because profiles are sent directly to Datadog without using the Datadog Agent, you must pass a valid [Datadog API key][3] by either:
+
+    - Adding `-Ddd.profiling.api-key.file=<PATH>`. This file should only contain the API key as a string.
+    - Setting the `DD_PROFILING_API_KEY=<DATADOG_API_KEY>` environment variable.
+
+5. Visualize your profiles in the [Datadog APM > Profiling page][4].
+
+**Note**:
+
+- The `-javaagent` needs to be run before the `-jar` file, adding it as a JVM option, not as an application argument. For more information, see the [Oracle documentation][5].
```

Comment:
An example of good and bad command lines might help clarify.

Reply:
Could you give some examples?

Reply:
> **Note**: `-javaagent` needs to be specified before `-jar` so it's treated as a JVM option and not an application argument.
> ```
> # Good:
> java -javaagent:dd-java-agent.jar ... -jar my-service.jar -more-flags
> 
> # Bad:
> java -jar my-service.jar -javaagent:dd-java-agent.jar ...
> ```